src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/Spec.java

```diff
@@ -42,19 +42,25 @@
     
     String[] config() default {};
     
+    /**
+     * The constant value designed for specification without related HTTP param. Const value can be a raw string or SpEL expression.
+     * <p>Supplying {@link #constVal} implicitly sets {@link #defaultVal} to empty
+     */
     String[] constVal() default {};
     
     /**
      * The default value to use as a fallback when the request parameter is
-     * not provided or has an empty value.
+     * not provided or has an empty value. Default value can be a raw string or SpEL expression.
```

Comment:
please add comment that SpEL support needs proper configuration as mentioned in the readme